src/main/webapp/js/instructorFeedbackEdit/visibilityOptions.js

```diff
@@ -53,38 +91,38 @@ function formatCheckBoxes() {
     // TODO: change class -> name?
     $('input[class*="answerCheckbox"]').change(function() {
         if (!$(this).is(':checked')) {
-            var visibilityOptionsRow = $(this).closest('tr');
-            visibilityOptionsRow.find('input[class*="giverCheckbox"]')
+            var editTabRows = $(this).closest('tr');
```

Comment:
Although the `$` wasn't present before, as we are changing names in this PR, maybe we could also prefix this with a `$` to distinguish jquery objects from regular variables